idseq_dag/steps/run_star.py

```diff
@@ -134,6 +155,10 @@ def run(self):
                                          output_gene_file)
                     command.move_file(gene_count_file, moved)
                     self.additional_files_to_upload.append(moved)
+                bam_file = os.path.join(tmp, "Aligned.out.bam")
+                if os.path.isfile(bam_file):
+                    moved = os.path.join(self.output_dir_local, self.bam_output)
+                    command.move_file(bam_file, moved)
```

Comment:
I wanted to make the output file name configurable by the DAG definition. Since there is no option I could find to change the output filename in the STAR manual I am moving it to the specified output file name.